create_pnor_image.pl

```diff
@@ -131,6 +131,7 @@
 $build_pnor_command .= " --binFile_SECBOOT $scratch_dir/secboot.bin.ecc";
 $build_pnor_command .= " --binFile_VERSION $openpower_version_filename";
 $build_pnor_command .= " --binFile_IMA_CATALOG $scratch_dir/ima_catalog.bin.ecc";
+$build_pnor_command .= " --binFile_WOFDATA $scratch_dir/wofdata.bin.ecc";
```

Comment:
You only added the partition to P9, doesn't this need to be in the P9-only branch?